### PR TITLE
added site entries

### PR DESCRIPTION
**Pull request by alkatoutl**

### CODE REVIEW

File: app/site/questions.js [hunk 1]
```diff
@@ -37,6 +37,18 @@ export const questionsParams = {
     component: "residents-with-disabilities",
     description: "Burnside is the neighborhood with the highest disability rate. What percentage of residents in Burnside, Chicago have an included disability?",
   },
+  "most-common-dropoff": {
```
Comment (vingkan) [inline app/site/questions.js:40]: Can you add taxi to these to distinguish from other modes of transit? Or are you planning to show multiple modes of transit?

Comment (alkatoutl) [reply app/site/questions.js:40]: Oh right I'll add taxi to them!

**Review by vingkan — APPROVED** [review]
Approved. I saw the sample ID for the component. Are you just adding site entries in this PR or also adding the question pages?

**Comment by alkatoutl** [comment]
> Approved. I saw the sample ID for the component. Are you just adding site entries in this PR or also adding the question pages?

just adding the site entries. my next PR will be implementing the frontend component!

**Pull request merged**